### DISCUSSION TITLE
Alunos SENAC-DF

### DISCUSSION THREAD

**Discussion by MARISTELAOLIVEIRA** [codewars/codewars.com#3042] [General]
Alunos, postem aqui seu Oi

**Comment by yuri735** [comment]
OI

**Reply by yuri735** [reply]
PROFESSORA
BOM TRABALHO

**Reply by antcarls** [reply]
Salve

**Reply by F1L1P323** [reply]
Fala professora tmj smp tlgd

**Reply by sabrinex0** [reply]
iae

**Reply by ryrg32** [reply]
Oie

**Reply by yuri735** [reply]
boa tarde

**Reply by HenriqueHHH** [reply]
OI

**Reply by J3FF3SON** [reply]
HELLO

**Comment by hobovsky** [comment]
Please note that Github Discussions are meant for discussing matters related to the  features and functionalities of the Codewars platform. Codewars Github is not meant as a place for community members to hang out, nor as a place to discuss general programming topics, or for general coding help, learning, etc.

I would suggest visiting Codewars Discord server and create your own thread there, for example in `#posts` section, where the students of your group can discuss their matters. Discord is also great because it has many active users and many channels related to potentially interesting topics.

I am closing this discussion as off-topic.
